blog/news/urls.py

```diff
@@ -1,13 +1,14 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
-from .views import NewsDetailView
+from .views import NewsDetailView, NewsList
 
 admin.autodiscover()
 
 urlpatterns = patterns('',
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^/?$', 'news.views.news_list', name='news-list'),
+    url(r'^/?$', NewsList.as_view(), name='news-list'),
     url(r'^news/(?P<slug>[\w\-_]+)/$', NewsDetailView.as_view(),
         name='news-detail'),
+    url(r'^(?P<page>[0-9]+)/$', NewsList.as_view(), name='news-list'),
```

Comment:
To mi się nie podoba. Póki co nie wiem, co to robi, ale dwa urle nie mogą mieć takich samych "name", ponieważ często w templatkach używamy urli po tym atrybucie właśnie. Jeśli ma ten url zostać, to "name" trzeba zmienić. Ale nie wiem, czy ten url jest konieczny. Chyba ta paginacja powinna działać normalnie bez tego, jeśli dobrze mi się wydaje, co to ma robić.